src/test/ui/proc-macro/attribute.rs

```diff
@@ -6,13 +6,13 @@
 extern crate proc_macro;
 
 #[proc_macro_derive]
-//~^ ERROR: attribute must be of form: #[proc_macro_derive(TraitName)]
+//~^ ERROR: attribute must be of the form `#[proc_macro_derive(...)]`
```

Comment:
Long term I wanted to attach some kind of grammar template to each built-in attribute, so it's fully checked.
That's certainly non-trivial, but perhaps attaching a plain string template that would be shown in case of error wouldn't be hard (even if tedious). I'll try.

Reply:
That long term plan sounds fantastic btw.